test/evergreen.yml

```diff
@@ -115,6 +115,15 @@ functions:
         rm -rf "wiredtiger"
         rm -rf "wiredtiger.tgz"
 
+  "checkpoint test":
+    command: shell.exec
+    params:
+      working_dir: "wiredtiger/build_posix/test/checkpoint"
+      script: |
+        set -o errexit
+        set -o verbose
+        ./t ${checkpointArgs} 2>&1
```

Comment:
Can you change it to checkpoint_args just to be consistent.

Reply:
Fixed